Source/Objects/Misc Objects/InFluxDB/ORInFluxDBModel.m

```diff
@@ -527,10 +527,10 @@ - (void) processStatusLogLine:(NSNotification*)aNote
                                   usingBlock:^(id value, NSRange range, BOOL *stop) {
         
         if([[value description] isEqualToString:@"_NSTaggedPointerColor"]){
-            NSString* s = [value description];
+            NSString* subString = [value description];
```

Comment:
Was shadowing a previous declaration of `s`. Don't know if this is a problem, changed to be safe.